scripts/check-pristine-state.js

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
```

Comment:
you calling this with node, therefore the hashbang isn't required? Otherwise we should name it like a true cli tool `check-pristine-state` and use like so `./check-pristine-state package-lock.json`